src/ore/src/lib.rs

```diff
@@ -33,3 +33,4 @@ pub mod sync;
 pub mod test;
 pub mod thread;
 pub mod tokio;
+pub mod vector;
```

Comment:
nit: these follow the modules in `std` directly so ideally this would be called `vec`.